ably/paginated_resource.go

```diff
@@ -40,8 +40,8 @@ type PaginatedResource struct {
 }
 
 // NewPaginatedResource returns a new instance of PaginatedResource
-// It needs to be a struct implementing ResourceReader which in our case is rest.Client.
-func NewPaginatedResource(typ reflect.Type, path string, params *config.PaginateParams,
+// It needs to be a struct implementing ResourceReader which in our case is rest.RestClient.
```

Comment:
It mentioned also non-existing `ResourceReader` interface. The `newPaginatedResource` is unexported now, so I've removed the doc.